### ISSUE TITLE
DeprecationWarning about time.clock removal in Python 3.8

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1872]
This occurs in the mock code on Python 3.7 with minimum package levels:
```
/Users/maiera/PycharmProjects/pywbem/pywbem/pywbem_mock/_wbemconnection_mock.py:3069: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
    'time': time.clock(),
```

**Comment by andy-maier** [comment]
The Python mock code uses the result of `time.clock()` to populate a time field in the enumeration context data. The time field is not yet used. A comment suggests that it can be used in the future for implementing a timeout at the pull level. Given that the base time of `time.clock()` is undefined at least on Windows, it seems that the new `time.perf_counter()` (added in Python 3.3) comes closest to it. Both return system wide elapsed time in fractional seconds.

**Comment by andy-maier** [comment]
Fixed in PR #1873 for master, and #1874 for 0.14.6.
